gatling-core/src/main/scala/io/gatling/core/feeder/ArrayBasedMap.scala

```diff
@@ -54,5 +62,15 @@ private[feeder] class ArrayBasedMap[K, +V](keys: Array[K], values: Array[V], ove
     }
   }
 
-  override def -(key: K) = throw new UnsupportedOperationException
+  override def -(key: K): Map[K, V] = {
```

Comment:
`Map.empty[K, V1] ++ this - key` is pretty similar (it might just add key and then remove it. I don't think we really care about performance here (or we'd have to benchmark) so I'd say we should go with the more simple form.

Reply:
Thanks